test/fixtures/vcenter-folder/variables.tf

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "vsphere_folder_config" {
+  description = "All Vsphere Folder Configs"
+  type        = any
+  default = {
+    "datacenter" : "Datacenter",
+    "path" : "folder-cft-test",
+    "type" : "vm",
+    "custom_attributes" : {
+      # "abc.xyz.Custom.Attribute" : "myvalue",
+      # "abc1.xyz1.Custom1.Attributes11" : "my2value"
+    },
+    "tags" : {
+      # "abc-l0" : {
+      #   "category_name" : "cat-abc"
+      # },
+      # "newtag-l0" : {
+      #   "category_name" : "cat-abc"
+      # }
+    },
+    "role_assignments" : {
+      # "permission1" : {
```

Comment:
I think for folder module we should still check if permissions are set correctly. Any reason you commented them out?

Reply:
@KonradSchieban : yep that can be done !  Since we were testing Folder creation just kept it to basics .We can definitely validate entity permissions too